e2e/container2consul.spec.ts

```diff
@@ -0,0 +1,55 @@
+import { test, expect } from '@playwright/test';
+import { startDockerCompose, stopDockerCompose } from './util';
+
+test.beforeAll(async () => {
+  await stopDockerCompose();
+  await startDockerCompose();
+  // Pause for 5 seconds to allow nginx test containers to start
+  await new Promise(r => setTimeout(r, 3000));
```

Comment:
```suggestion
  await new Promise(r => setTimeout(r, 5000));
```

Reply:
@kdaud  is this based on your tests locally?

Reply:
Setting 3000 milliseconds equates to 3 seconds, and the comment indicates that a pause of 5 minutes is required to allow the Nginx test containers to start.